geth/params/config.go

```diff
@@ -529,21 +558,32 @@ func (c *NodeConfig) updateBootClusterConfig() error {
 		BootNodes []string `json:"bootnodes"`
 	}
 	type clusterConfig struct {
-		NetworkID   int              `json:"networkID"`
-		GenesisHash string           `json:"genesisHash"`
-		Prod        subClusterConfig `json:"prod"`
-		Dev         subClusterConfig `json:"dev"`
+		NetworkID int              `json:"networkID"`
+		Prod      subClusterConfig `json:"prod"`
+		Dev       subClusterConfig `json:"dev"`
 	}
 
-	chtFile, err := static.Asset("config/staticpeers.json")
-	if err != nil {
-		return fmt.Errorf("staticpeers.json could not be loaded: %s", err)
+	var chtFile []byte
```

Comment:
This naming has left from the previous refactoring. Something like `clusterData` should be more appropriate.

Reply:
Good opportunity, will do.